app/lib/server/server.go

```diff
@@ -144,10 +146,17 @@ func InteractiveConfig(binaryName string) {
 
 	// create the keys
 	privStr, pubStr := createKeyPair()
+	ips, err := net.LookupIP(publicAddress.Host())
+	log.ErrFatal(err)
+	cert, key, err := network.NewCertKey(network.NewTLSCert(big.NewInt(1), "ch", "epfl", "dedis",
```

Comment:
One thing I don't like about user-input is that it's difficult to script it. Of course you can use `echo -e "arg1\narg2\narg3"`, but then you cannot change the order of the inputs anymore because you would break the scripts.